checkov/gitlab/dal.py

```diff
@@ -0,0 +1,69 @@
+import os
+
+from checkov.common.runners.base_runner import strtobool
+from checkov.common.vcs.base_vcs_dal import BaseVCSDAL
+
+
+class Gitlab(BaseVCSDAL):
+    def __init__(self):
+        super().__init__()
+        self.configure_gitlab_conf_dir()
```

Comment:
@schosterbarak Maybe make the `BaseVCSDAL` call the `setup_conf_dir` method, make it abstract and enforce it is overridden in the children of `BaseVCSDAL`? Just like `discover`